bears/js/HappinessLintBear.py

```diff
@@ -20,5 +20,14 @@ class HappinessLintBear:
     CAN_DETECT = {'Syntax'}
 
     @staticmethod
-    def create_arguments(filename, file, config_file):
+    def create_arguments(
+            filename, file, config_file,
+            use_spaces: bool = False):
+
+        if (use_spaces):
+            raise ValueError(
+                'Set "use_spaces = False" for HappinessLintBear'
+                'to avoid conflicts with other bears'
+            )
```

Comment:
if you're following PEP8 standards, then check the indentation here

Reply:
It also follows PEP8.

Reply:
sorry then I think it's my bad here

other people will see to it then 😛

Reply:
:P nevermind. Btw git mate is always there for such indentation matters :)